service/src/test/java/bio/terra/lz/futureservice/pact/consumer/SamServiceConsumerPactTest.java

```diff
@@ -54,19 +50,26 @@ public RequestResponsePact statusApiPact(PactDslWithProvider builder) {
         .toPact();
   }
 
-  @Pact(consumer = "terra-landing-zone-service", provider = "sam")
-  public RequestResponsePact resourceExistingPermissionV2Pact(PactDslWithProvider builder) {
-    return builder
-        .given("permission exists")
-        .uponReceiving("a request to check permissions")
-        .path("/api/resources/v2/%s/%s/action/%s".formatted(RESOURCE_TYPE, RESOURCE_ID, ACTION))
-        .method("GET")
-        .willRespondWith()
-        .status(200)
-        .body("true")
-        .toPact();
-  }
+  /*
+    Tests are commented out because just marking them as disabled or with @PactIgnore wasn't working,
+    and they were still being added to the contract.
+
 
+    @Disabled
```

Comment:
Why do these need to be disabled?

Reply:
They don't correspond with any existing provider state in Sam - so they'll always fail.